src/App.tsx

```diff
@@ -17,18 +75,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onSearch={handleSearchQueryChange}
+                onSelectFilter={handleStatusFilterChange}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {filteredTodos ? (
+                <TodoList
+                  todos={filteredTodos}
+                  selectedTodoId={selectedTodoId}
+                  onSelect={handleSelectTodo}
+                />
+              ) : error ? (
+                <p>Something went wrong!</p>
+              ) : (
+                <Loader />
+              )}
```

Comment:
please don't nest ternaries

Reply:
I am not sure how to format it, look into the version that I've just commited